scripts/zones/Upper_Jeuno/npcs/_6s1.lua

```diff
@@ -1,106 +1,124 @@
------------------------------------
+----------------------------------------------------------------------------------------------------------------------------
```

Comment:
why are you adding this huge dashed line? Stick with the format we already use in literally every file in the project. I am not a fan of unique snowflakes.